source/third_party.adoc

```diff
@@ -0,0 +1,57 @@
+= Third Party Library Dependencies
+
+Adding third party libraries to be bundled in your plugin is something that should be done rarely and only by seeking feedback on the mailto:jbosstools-dev@lists.jboss.org[jbosstools-dev@list.jboss.org] mailing list. 
+
+== Do not push libraries into github
+
+Adding third-party libraries to your github repository makes your repository very large and slow to download. This is not a generally approved method of adding dependencies to your plugin. In extreme circumstances it may be approved on a case-by-case basis, and only after discussion on the mailto:jbosstools-dev@lists.jboss.org[jbosstools-dev@list.jboss.org] mailing list. 
+
+== So how do I add third party dependencies?
+
+The proper way to bundle third-party libraries with your plugin is to add those dependencies to your pom.xml. This will ensure that those libraries are fetched during a build. The below example will fetch `jboss-logging-3.1.2.GA` and put it inside the `lib` folder inside your plugin. Add code similar to this example to your plugin's pom.xml:
```

Comment:
the *proper* way is to see if they can be made part of the TP, if not, then part of Locus and only then if not possible add as a embedded jar dependency.